<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>login_ext_form</class>
 <widget class="QWidget" name="login_ext_form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LoginDlgExt</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDBServer" />
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Database:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDBName" />
   </item>
   <item>
    <widget class="QLabel" name="lDBPort" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDBPort" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../xgis.qrc" />
 </resources>
 <connections/>
</ui>
